extensions/datastores/accumulo/src/main/java/mil/nga/giat/geowave/datastore/accumulo/util/AccumuloUtils.java

```diff
@@ -103,6 +103,10 @@
 
 	public static Range byteArrayRangeToAccumuloRange(
 			final ByteArrayRange byteArrayRange ) {
+		if (byteArrayRange.isSingleValue()) {
```

Comment:
out of curiousity, did this relate to a bug encountered? ie. how did you come across this change?